core/vtkOsmLayer.cxx

```diff
@@ -160,7 +167,7 @@ void vtkOsmLayer::Update()
 
   this->AddTiles();
 
-  this->Superclass::Update();
+  this->Superclass::Update(); // redundant isn't it ????
```

Comment:
Is the comment still relevant? I don't _think_ there is anything redundant going on here.

Reply:
It's not relevant !